e2e-tests/pipeline_validation.sh

```diff
@@ -195,6 +195,18 @@ function fhir_source_query() {
 #   OPENMRS
 #################################################
 function test_parquet_sink() {
+  # This global variable is hardcoded to validate the View record count
+  # which can greater than the number of Resources in the source FHIR
+  # Server due to flattening
+  PATIENT_VIEW_ROWCOUNT=0
+  OBS_VIEW_ROWCOUNT=284925
+  if [[ -n ${OPENMRS} ]]; then
+    PATIENT_VIEW_ROWCOUNT=110
```

Comment:
You could also update the `OBS_VIEW_ROWCOUNT` similar to `PATIENT_VIEW_ROWCOUNT` to have the right value for `openmrs` fhir source, post which the additional if conditions at lines 241 and 247 become irrelevant and can be merged.

Reply:
Done.